### PR TITLE
Simplifies the example codes for glossary entries of boundary_id and manifold_id

### LINKED ISSUE #9319: Improve the example codes in the documentation module/glossary entry for boundary indicator id and manifold_id
Currently, we have rather convoluted code explaining the setting of boundary indicator id in the [documentation](https://dealii.org/developer/doxygen/deal.II/DEALGlossary.html#GlossBoundaryIndicator).

This is related to #9227 

I will provide a PR to simplify this code and close this issue.

### PR DESCRIPTION
**Pull request by krishnakumarg1984**
Fixes #9319

### CODE REVIEW

File: doc/doxygen/headers/glossary.h [hunk 1]
```diff
@@ -327,14 +327,9 @@
...
- *   for (typename Triangulation<dim>::active_cell_iterator
- *          cell = triangulation.begin_active();
- *        cell != triangulation.end();
- *        ++cell)
- *     for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
- *       if (cell->face(f)->at_boundary())
- *         if (cell->face(f)->center()[0] == -1)
- *           cell->face(f)->set_boundary_id (42);
+ * for (auto &face : triangulation.active_face_iterators())
+     if (face->center()(0) == -1)
```
Comment (bangerth) [inline doc/doxygen/headers/glossary.h]: You still need the check for whether the face is at the boundary.

**Comment by krishnakumarg1984** [comment]
I am absolutely confused with the git operations here. Anyway, shall wait for review from the moderators.

**Comment by bangerth** [comment]
There's a review already up above -- you accidentally dropped the `if (face->at_boundary())` check. Other than that, this is good to go.

**Comment by krishnakumarg1984** [comment]
I have to close this PR due to getting too confused with git branch/merge issues. Have addressed the reviewer comments in #9379

**Comment by peterrum** [comment]
Please do not close PRs just so and open another one with the same name. Keeping the whole discussion together in one place is important for documentation purposes.

**Comment by krishnakumarg1984** [comment]
Sorry about that. I got really confused with the git operations to correctly update the PR with required fixes.

**Pull request closed without merging**